packages/contracts/test/contracts/OVM/precompiles/OVM_SequencerEntrypoint.spec.ts

```diff
@@ -2,11 +2,13 @@ import { expect } from '../../../setup'
 
 /* External Imports */
 import { waffle, ethers } from 'hardhat'
-import { ContractFactory, Wallet, Contract } from 'ethers'
+import { ContractFactory, Wallet, Contract, BigNumber } from 'ethers'
 import { smockit, MockContract } from '@eth-optimism/smock'
+import { fromHexString, toHexString } from '@eth-optimism/core-utils'
+import { ecrecover, publicToAddress } from 'ethereumjs-util'
```

Comment:
Is it possible to use https://docs.ethers.io/v5/api/utils/address/#utils-recoverAddress in favor of adding a new dependency?

Reply:
Done